src/core/Akka.Streams/Dsl/Source.cs

```diff
@@ -930,7 +930,7 @@ public static class Source
         /// <param name="close">function that closes resource</param>
         /// <returns>TBD</returns>
         public static Source<T, NotUsed> UnfoldResourceAsync<T, TSource>(Func<Task<TSource>> create,
-            Func<TSource, Task<Option<T>>> read, Func<TSource, Task> close)
+            Func<TSource, Task<Option<T>>> read, Func<TSource, Task<Done>> close)
```

Comment:
Breaking change, but the signature was wrong.

Reply:
Not a big deal, IMHO